spec/rails_helper.rb

```diff
@@ -11,7 +11,13 @@
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
-  config.fixture_path = "#{::Rails.root}/test/fixtures"
+  # TestFixtures#fixture_path was removed in Rails 7.2, use fixture_paths instead.
+  if config.respond_to?(:fixture_paths)
+    config.fixture_paths = ["#{::Rails.root}/test/fixtures"]
+  else
+    config.fixture_path = "#{::Rails.root}/test/fixtures"
+  end
```

Comment:
Since I don’t know the minimum Redmine version targeted by the tests running on CircleCI, I implemented it with Rails 7.0 and earlier (Redmine 5.1 and earlier) in mind. If the target is Rails 7.1 or later only, the else clause should not be necessary.